apps/prairielearn/src/middlewares/authn.js

```diff
@@ -110,7 +118,10 @@ module.exports = asyncHandler(async (req, res, next) => {
   }
   if (authnData == null) {
     // We failed to authenticate.
-    //
+
+    // Clear the pl_authn cookie in case it was bad
+    res.clearCookie('pl_authn');
```

Comment:
This change wasn't necessary; it just seemed like a good idea to unconditionally clear that authentication cookie if we failed to decode it.